packages/ui5-library-writer/src/index.ts

```diff
@@ -6,6 +6,8 @@ import cloneDeep from 'lodash/cloneDeep';
 import type { UI5LibConfig } from './types';
 import { enableTypescript } from './options';
 import { mergeWithDefaults } from './data';
+import type { UI5 } from '@sap-ux/ui5-application-writer';
+import { getTemplateVersionPath } from '@sap-ux/ui5-application-writer';
```

Comment:
`@sap-ux/ui5-application-writer` shouldn't be a dependency for `ui5-library-writer` (Seems is is unused dependency before this and should be removed from the package.json. 
getTemplateVersionPath and UI5 should moved to common module or duplicated.